app/lib/screens/HomeScreens/ChatScreen.dart

```diff
@@ -118,6 +119,29 @@ class _ChatScreenState extends State<ChatScreen> {
     );
   }
 
+  Future<FfiBufferUint8> _userAvatar(String userId) async {
+    final member = await widget.room.getMember(userId);
+    return member.avatar();
+  }
+
+  Widget _avatarBuilder(String userId) {
+    return GetBuilder<ChatController>(
+      id: 'Avatar',
+      builder: (ChatController controller) {
+        return Padding(
+          padding: const EdgeInsets.only(right: 10),
+          child: CustomAvatar(
+            avatar: _userAvatar(userId),
+            displayName: null,
+            radius: 15,
```

Comment:
Also the image avatar radius appear smaller and text avatar appears bigger. Is there a way to make it even to image avatar size?

Reply:
What if this code should work on desktop or web, not just mobile?

Reply:
On mobile, it looks weirdly. Like images avatars appear smaller and text one bigger. Not a good looking conversation UI. :)

Reply:
Anyways we should do something about text avatars size. They should be even depending upon platforms.

Reply:
Is it impossible to resize image on flutter side?